saul-core/src/main/scala/edu/illinois/cs/cogcomp/saul/classifier/ClassifierUtils.scala

```diff
@@ -19,48 +19,48 @@ object ClassifierUtils extends Logging {
     def apply[T <: AnyRef](c: (Learnable[T], Iterable[T])*) = {
       c.foreach {
         case (learner, trainInstances) =>
-          logger.info(evalSeparator)
-          logger.info("Training " + learner.getClassSimpleNameForClassifier)
+          println(evalSeparator)
```

Comment:
I changed it because it was messing up the formatting of the output results. But now in retrospect, I think only the ones related to testing was needed. I returned back the rest.